hashicorp/boundary-ui
# Icu 13880 create manage scopes route and view with keyword values

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-13880

# Description





## Screenshots (if appropriate)


https://github.com/hashicorp/boundary-ui/assets/107949262/22e2451d-eaf7-43fa-97d2-b00cd0061446


## How to Test

1. Navigate to a global level role
2. Click manage dropdown
3. Click Manage scopes
4. User should be able to toggle **this**, **children**, and **descendants** & save & cancel changes
5. Navigate to org level role
6. Click manage dropdown
7. Click Manage scopes
8. User should be able to toggle **this**, and **children** & save & cancel changes
9. Navigate to project level role
10. Click manage dropdown
11. Manage scopes link should not be visible.

## Checklist
